knox-homepage-ui/homepage/app/generalProxyInformation/general.proxy.information.component.ts

```diff
@@ -0,0 +1,111 @@
+                        <tr><th colspan="2" align="center">General Proxy Information</th></tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td>Knox Version</td>
+                            <td>{{ getVersion() }}</td>
+                        </tr>
+                        <tr>
+                            <td>TLS Public Certificate PEM file</td>
+                            <td><a href="{{ getPublicCertPemPath() }}" target="_blank">Click here to download</a></td>
+                        </tr>
+                        <tr>
+                            <td>TLS Public Certificate JKS file</td>
+                            <td><a href="{{ getPublicCertJksPath() }}" target="_blank">Click here to download</a></td>
+                        </tr>
+                        <tr>
+                            <td>Admin UI URL</td>
+                            <td><a href="{{ getAdminUiUrl() }}" target="_blank">{{ getAdminUiUrl() }}</a></td>
+                        </tr>
+                        <tr>
+                            <td>
+                                Admin API
+                                <span class="inline-glyph glyphicon glyphicon-info-sign btn btn-xs"
+                                title="{{ getAdminApiDescription() }}" data-toggle="tooltip"></span>
+                            </td>
+                            <td>
+                                <a href="{{ getAdminApiUrl() }}" target="_blank">{{ getAdminApiUrl() }}</a>
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+    `,
+    providers: [HomepageService]
+})
+
+export class GeneralProxyInformationComponent implements OnInit {
+
+    generalProxyInformation: GeneralProxyInformation;
+
+    constructor(private homepageService: HomepageService) {}
+
+    getVersion() {
+        if (this.generalProxyInformation) {
+            return this.generalProxyInformation.version;
+          }
+          return '';
+    }
+    getPublicCertPemPath() {
+      if (this.generalProxyInformation) {
+        return this.generalProxyInformation.publicCertPemPath;
+      }
+      return '';
+    }
+
+    getPublicCertJksPath() {
+        if (this.generalProxyInformation) {
+          return this.generalProxyInformation.publicCertJksPath;
+        }
+        return '';
+    }
+
+    getAdminUiUrl() {
+        if (this.generalProxyInformation) {
+            return this.generalProxyInformation.adminUiUrl;
+          }
+          return '';
+    }
+
+    getAdminApiDescription() {
+       return 'Knox provides a REST API which allows end-users executing CRUD operations on topologies/shared-porviders/descriptors/'
+              + 'service definitions as well as fetching the Knox version.';
+    }
+
+    getAdminApiUrl() {
+        return 'https://knox.apache.org/books/knox-1-4-0/user-guide.html#Admin+API';
```

Comment:
should just be `https://knox.apache.org/books/knox-1-4-0/user-guide.html#Admin+API` Looks like smart dashes in the line.

In either case since we are printing the version of Knox - it should be possible to dynamically created this url based on `this.generalProxyInformation.version`

Reply:
Done: admin API book URL is calculated on the back-end - based on the build version - and being sent as part of the general proxy information payload.